### PR TITLE
sage/connect-and-interact-with-celo-using-web3modal

### PR DESCRIPTION
**Pull request by bobeu**
Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code, with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani ```

### CODE REVIEW

File: blog/10-03-2023-connect-and-interact-with-celo-using-web3modal/connect-and-interact-with-celo-using-web3modal.md [hunk 1]
```diff
@@ -0,0 +1,501 @@
...
+  - name: Isaac Jesse
+    title: Web3, Smart Contract Developer
+    url: https://github.com/bobeu
+    image_url: https://github.com/bobeu.png
+tags: [advanced, celosage, solidity, react]
+hide_table_of_contents: true
+slug: /tutorials/connect-and-interact-with-celo-using-webmodal
+---
+
+![header](../../src/data-tutorials/showcase/advanced/connect-and-interact-with-celo-using-web3modal.png)
```
Comment (developerkunal) [inline blog/10-03-2023-connect-and-interact-with-celo-using-web3modal/connect-and-interact-with-celo-using-web3modal.md:14]: `../../src/data-tutorials/showcase/advanced/connect-and-interact-with-celo-using-webmodal.png`
Change the Image Url To This

File: src/data-tutorials/users.tsx [hunk 2]
```diff
@@ -2070,11 +2070,18 @@ const Users: User[] = [
     tags: ['javascript', 'react', 'beginner', 'celo'], 
   },
   {
+    title: 'Connect and interact with Celo using web3modal',
+    description: 'Alternative method of connecting to Celo using web3React',
+    preview: require('./showcase/advanced/connect-and-interact-with-celo-using-web3modal.png'),
```
Comment (developerkunal) [inline src/data-tutorials/users.tsx:2075]: `./showcase/advanced/connect-and-interact-with-celo-using-webmodal.png`
change the url here

File: blog/10-03-2023-connect-and-interact-with-celo-using-web3modal/connect-and-interact-with-celo-using-web3modal.md (hunk 1 above)
Comment (bobeu) [reply blog/10-03-2023-connect-and-interact-with-celo-using-web3modal/connect-and-interact-with-celo-using-web3modal.md]: Resolved

File: src/data-tutorials/users.tsx (hunk 2 above)
Comment (bobeu) [reply src/data-tutorials/users.tsx:2075]: This path seems to be correct. Could you please check again?

Comment (developerkunal) [reply src/data-tutorials/users.tsx:2075]: If you check correctly you used web3modal but in image name its webmodal.png

Comment (developerkunal) [reply src/data-tutorials/users.tsx:2075]: @bobeu

File: blog/10-03-2023-connect-and-interact-with-celo-using-web3modal/connect-and-interact-with-celo-using-web3modal.md [hunk 3]
```diff
@@ -0,0 +1,501 @@
...
+  - name: Isaac Jesse
+    title: Web3, Smart Contract Developer
+    url: https://github.com/bobeu
+    image_url: https://github.com/bobeu.png
+tags: [advanced, celosage, solidity, react]
+hide_table_of_contents: true
+slug: /tutorials/connect-and-interact-with-celo-using-webmodal
+---
+
+../../src/data-tutorials/showcase/advanced/connect-and-interact-with-celo-using-webmodal.png
```
Comment (developerkunal) [inline blog/10-03-2023-connect-and-interact-with-celo-using-web3modal/connect-and-interact-with-celo-using-web3modal.md:14]: `here you still need the header`

**Comment by developerkunal** [comment]
@bobeu Update the PR Description 

like this one 
Thanks for submitting this tutorial for Celo Sage
Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

 My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
 The title of the Pull Request matches the title of my tutorial.
 I've followed the tutorial template structure.
 I've proofread the tutorial for spelling and grammar mistakes.
 I've thoroughly explained the code, with inline code comments and text explanations.
 I've run a linter & prettifier to format my code blocks.
 I've run the tutorial to test it and the code works.
NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
Reviewers - @joenyzio & @viral-sangani

**Review by developerkunal — APPROVED** [review]
Approved @Ishanpathak1

**Review by Ishanpathak1 — APPROVED** [review]
Approved

**Pull request merged**